### PR TITLE
Adding docker VACOLS

### PR DESCRIPTION
**Pull request by mdbenjam**
Step one in Dockerizing VACOLS.

To review this PR you'll need:
1) Docker installed
2) Docker logged in and this download approved: https://store.docker.com/images/oracle-database-enterprise-edition/

**Test Plan**
- [x] Run `rake local_vacols:setup` to start the DB.
- [x] Run `rails c` and try to connect by running `VACOLS::Case.create(bfkey: "1")` and then `VACOLS::Case.find(1)

### CODE REVIEW

**Comment by sunil-sadasivan** [comment]
Niice, good call using rake to abstract pure docker-compose commands.

File: lib/tasks/local_vacols.rake [hunk 1]
```diff
@@ -0,0 +1,64 @@
+namespace :local_vacols do
+  desc "Starts and sets up a dockerized local VACOLS"
+  task setup: :environment do
+    Dir.chdir(Rails.root.join("vacols")) do
+      puts "Removing existing volumes"
+      `docker-compose down -v`
+      puts "Starting database"
+      `docker-compose up &> '../tmp/vacols.log' &`
```
Comment (sunil-sadasivan) [inline lib/tasks/local_vacols.rake]: Good call to output log here.  We should indicate in the stdout log info that the verbose output is being logged to `./tmp/vacols.log`  I had to view the source to understand how I could see the docker output.

**Comment by mdbenjam** [comment]
Yay! Thanks for giving it a go.

Hmmm I'd installed the OCI stuff so long ago, I'd forgotten how much of a pain it is. I actually think a good part of the team already has it installed since a lot of people have connected to staging in the past. If we want to automate it... I'm thinking upload the client directories to S3, and have a script to download them and do all the brew stuff. Though that means people would need to have access to AWS to use that. I'm wondering if that's worth it? Another approach is have people still download it manually, but have a rake job to do the installation steps. Then we can have people sign in with a common Caseflow account for both docker and getting the OCI.

**Comment by sunil-sadasivan** [comment]
> Though that means people would need to have access to AWS to use that.

Not if we make the bucket a public one.  (Though this probably violates the oracle license agreement 😞)

Now that I think about it, it might be ok for us to just add readme instructions to install this.  So I'm ok with not making this smooth for now.  However just know it'll break the `bundle install` for devs so we'll need this communicated to the team.

**Comment by mdbenjam** [comment]
Yeah making it public is probably a license violation. We could play the "how quickly can we get a cease and desist letter from Oracle game"!

The README should have these instructions in the "connecting to staging section". I'll move them to the general setup steps. I just put a docker account with the license accepted that we can all share. I'll add an error state in the rake commands if you're not logged in.

**Comment by sunil-sadasivan** [comment]
Awesome, sounds good @mdbenjam!

File: vacols/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+version: '3'
+services:
+  vacols-db:
+    build: .
+    container_name: VACOLS_DB
+    image: vacols_test
```
Comment (sunil-sadasivan) [inline vacols/docker-compose.yml]: Would it be better to rename this `vacols_db`

**Pull request merged**